backend/apps/matching/urls.py

```diff
@@ -56,7 +56,7 @@
     ####################################
     path("profile_staff", views.StaffProfileView.as_view(), name="staff_profile"),
     path(
-        "staff/<p:p_type>/approve",
+        "staff/<p:p_type>/approve/",
```

Comment:
It strangely stopped working without the `/`

Reply:
One more reason for #130! :tada: :wink:

Reply:
And for #157 now :grin: